packages/angular_devkit/schematics/src/index.ts

```diff
@@ -22,6 +22,7 @@ export * from './rules/random';
 export * from './rules/schematic';
 export * from './rules/template';
 export * from './rules/url';
+export * from './rules/rename';
```

Comment:
This change adds an element to the public API surface.  This should be avoided if possible since each addition adds to the long term sustainment cost.

This is also why the  bazel build CI is failing.  It has a public API check.  New public API baselines need to be accepted before the check passes.  This helps ensure that new additions were not accidental.

Reply:
Instead of rename now I used forEach that was already available in public api